src/OrchardCore/OrchardCore.Data.YesSql/OrchardCoreBuilderExtensions.cs

```diff
@@ -166,6 +166,7 @@ private static IConfiguration GetStoreConfiguration(IServiceProvider sp, YesSqlO
                 QueryGatingEnabled = yesSqlOptions.QueryGatingEnabled,
                 ContentSerializer = new PoolingJsonContentSerializer(sp.GetService<ArrayPool<char>>()),
                 TableNameConvention = yesSqlOptions.TableNameConvention,
+                IdentityColumnSize = yesSqlOptions.UseLegacyIdentityColumn ? IdentityColumnSize.Int32 : IdentityColumnSize.Int64
```

Comment:
I would suggest you use the same enum used by YesSql `IdentityColumnSize` instead of using a boolean `UseLegacyIdentityColumn`  maybe at somepoint YesSql will support other enum values like string.

Reply:
It's column size not type, I think it's fine to stay with a boolean value, but I never mind to use the `IdentityColumnSize `

Reply:
yes but it is an enum value which can have more than 2 values. Maybe it does not have the best name in YesSql but it should also be an enum in OC not bool.

Reply:
Ok the, `IdentityColumnType` looks much better